File: concrete/src/main/java/pcb/uwu/unit/quantity/AmpereUnit.java
package pcb.uwu.unit.quantity;

import pcb.uwu.units.quantity.ElectricCurrentUnit;

import java.util.function.Function;

public class AmpereUnit extends ElectricCurrentUnit {

	public static final AmpereUnit AMPERE = new AmpereUnit();

	private AmpereUnit() {
		super("A", "ampere", "amperes",
				Function.identity(),
				Function.identity());
	}
}

File: implementation/src/main/java/pcb/uwu/unit/base/AmpereUnit.kt
package pcb.uwu.unit.base

import pcb.uwu.util.identity

object AmpereUnit
    : ElectricCurrentUnit(symbol = "A",
                          singularName = "ampere",
                          pluralName = "amperes",
                          translationToCanonical = identity(),
                          translationFromCanonical = identity())